rocAL_pybind/amd/rocal/plugin/generic.py

```diff
@@ -218,6 +218,57 @@ def __init__(self,
                                                           multiplier=pipe._multiplier, offset=pipe._offset, display=display, device=device, device_id=device_id)
 
 
+class ROCALNumpyIterator(object):
```

Comment:
Do we need a separate iterator, why not use the generic iterator for this
And we can check if it is numpy reader and only return the output list

In generic iterator only the last batch check is missing I feel that can be added please check

Reply:
In numpy reader, there is no labels so we return only images. In generic iterator, even though label reader is not present, it returns label tensor which is set as None

Reply:
But we can have if conditions to check if it is Numpy reader right?
I feel that the use of another iterator adds extra code